tests/Unit/Entity/AssignmentTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
please move it after the copyright block :)

Reply:
It's always before the copyright block in the whole project. Are you sure ?

Reply:
Good question. The point is that, as I see in backend channel - we had an agreement to have `declare` block after the copyright statement (messages from 03/26 and 03/27). Maybe it's not a good point of time to change this for the whole project, but we can stick to this for the new files, no? :upside_down_face: